ppr-ui/src/store/state/state-model.ts

```diff
@@ -29,6 +29,7 @@ export const stateModel: StateModelIF = {
       lastUpdateDateTime: null
     },
     expiryDate: '',
+    surrenderDate: '',
```

Comment:
surrenderDate is now defined in registration and registration.lengthTrust?

Reply:
yes, I'm not sure why I did that. I have removed it